README.md

```
@@ -1,202 +1,248 @@
+Manage git repositories with directory conventions.
 
 ## Why?
 
-How do you manage git repository?
-
-Maybe you create a directory and clone to it. However if you want to clone repository that has same name? Or Do something in every directory like `clean`?
-
-`Projj` provide a structure making it easy.
+Every git repo gets a predictable home based on its URL:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)